packages/sdk-ts/src/client/indexer/types/explorer-rest.ts

```diff
@@ -225,3 +225,19 @@ export interface CW20BalanceExplorerApiResponse {
 }
 
 export { BaseTransaction }
+
+export interface BankTransferFromExplorerApiResponse {
+  sender: string
+  recipient: string
+  amounts: { denom: string; amount: string }[]
+  block_number: number
+  block_timestamp: string
+}
+
+export interface BankTransfer {
```

Comment:
can you move `BankTransfer` to `sdk-ts/src/client/indexer/types/explorer.ts`?

also, let's use `import { Coin } from '@injectivelabs/ts-types'` to replace the `{ denom: string; amount: string }`